pyclam/manifold.py

```diff
@@ -369,6 +375,7 @@ def __init__(self, *clusters):
 
         # self.clusters is a dictionary of the clusters in the graph and the connected component subgraph that the cluster belongs to.
         self.clusters: Dict[Cluster: 'Graph'] = {c: None for c in clusters}
+        self.count = 0
```

Comment:
What does this do

Reply:
ugh I've gotten rid of this line so many times